src/cleanup_rules/swift/rules.toml

```diff
@@ -785,7 +790,7 @@ replace = ""
 is_seed_rule = false
 
 [[rules.filters]]
-enclosing_node = "(class_declaration) @cd"
+enclosing_node = "[(class_declaration) (init_declaration)] @cd"
```

Comment:
Is this change related to this PR?

Reply:
This change is not directly related to the PR but solves the issue of inlining variables declared in init scope in the class itself. Earlier fix provided in #471 was not enough. This separates the flow and ensures things work correctly. Since this was just a one line change, I just included here.

Reply:
update the PR description for the PR, and mark this issue as related (And close it once u land the PR)